frontend/model/contracts/group.js

```diff
@@ -1441,6 +1446,15 @@ sbp('chelonia/defineContract', {
           ['gi.contracts/group/releaseDeletedChatRoom', state.chatRooms[data.chatRoomID].members, data.chatRoomID]
         )
         Vue.delete(state.chatRooms, data.chatRoomID)
+      },
+      sideEffect ({ data, contractID, innerSigningContractID }) {
+        sbp('okTurtles.events/emit', DELETED_CHATROOM, { groupContractID: contractID, chatRoomID: data.chatRoomID })
+        const { identityContractID } = sbp('state/vuex/state').loggedIn
```

Comment:
The `DELETED_CHATROOM` listener is defined to also take an `identityContractID`:

```js
{ identityContractID, groupContractID, chatRoomID }
```

So should this include `identityContractID: innerSigningContractID` here?

Reply:
It's optional, but, yes, I think it could be added.

Reply:
It should not be used when deleting a chatroom. If it's set, it'll skip the handler unless the user deleting it is the same as the currently logged in user.

Reply:
Then could you please add a comment saying that the `identityContractID` is intentionally left out?